src/styles/shared/_interaction-state.scss

```diff
@@ -24,3 +24,16 @@
   }
   background-image: $backgrounds;
 }
+
+@mixin list-single-selection-state {
+  background: var(--p-interactive);
+  content: '';
+  position: absolute;
+  top: 0;
+  left: -1 * spacing(tight);
```

Comment:
This here is the critical piece - `spacing(tight)` needs to match exactly what the `padding-left/right` is on the consuming "list" _(`ActionList/OptionList`)_.

Right now, coincidentally, they both use `spacing(tight)`. Just to be a little more explicit / future-proof, perhaps we make this an argument?

```scss
@mixin list-selected-indicator($offset: spacing(tight)) {
  left: -1 * $offset;
}
```